addons/external-dns/external-dns.yaml

```
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: external-dns
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.7.0-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.7.0-2"
     appversion.kubeaddons.mesosphere.io/external-dns: "0.7.0"
     values.chart.helm.kubeaddons.mesosphere.io/external-dns: "https://raw.githubusercontent.com/helm/charts/b9191a5b526d4988b7938be2049370a1687037a0/stable/external-dns/values-production.yaml"
```

```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/external-dns: "https://raw.githubusercontent.com/bitnami/charts/9eca03cb2f4b9d4a5b7e877711f810f9151ea2a3/bitnami/external-dns/values.yaml"
```
(I think that's the right values for 2.20.4?)